types/node/worker_threads.d.ts

```diff
@@ -17,41 +19,49 @@ declare module "worker_threads" {
 
     class MessagePort extends EventEmitter {
         close(): void;
-        postMessage(value: any, transferList?: Array<ArrayBuffer | MessagePort>): void;
+        postMessage(value: any, transferList?: Array<ArrayBuffer | MessagePort | FileHandle>): void;
```

Comment:
According to docs also `KeyObject` is allowed in `transferList`.

Reply:
I think the same change should be also applied to `WorkerOptions.transferList`. Maybe use a dedicated type to avoid duplication?

Reply:
From history:
> Added `KeyObject` to the list of cloneable types.

@Flarna so `KeyObject` is allowed in `value`, not in `transferList`.

From docs:
> `transferList` may be a list of `ArrayBuffer`, `MessagePort` and `FileHandle` objects. 

also
> `value` may not contain native (C++-backed) objects other than `MessagePort`s, `FileHandle`s, and `KeyObject`s.

Reply:
Yes, sorry. Misread the docs.

Reply:
It should still probably use a type alias:
```ts
type TransferListItem = ArrayBuffer | MessagePort | FileHandle;
```

```suggestion
        postMessage(value: any, transferList?: TransferListItem[]): void;
```

Reply:
It looks better 👍 I have updated PR

Reply:
I still think that `WorkerOptions.transferList` should be adapted to be of type `TransferListItem[]`

Reply:
@Flarna should I hold on approving this until `WorkerOptions.transferList` has been updated?

Reply:
I would prefer to add it here but it's also ok to do it in a followup PR.